Cargo.toml

```diff
@@ -146,10 +146,13 @@ all = [
   "memcached",
   "gcs",
   "azure",
+  "buildless",
   "gha",
   "webdav",
 ]
 azure = ["opendal", "reqsign"]
+buildless = ["opendal", "redis", "webdav"]
```

Comment:
since buildless implements webdav and redis, this code simply reuses those adapters to reduce code surface